index.html

```diff
@@ -70,17 +70,18 @@ <h3 class="card-head__heading">Mr. Eat</h3>
         <div class="card__card-head">
           <img
             class="card-head__img"
-            src="https://images.unsplash.com/photo-1542528180-0c79567c66de?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=80"
+            src="https://images.unsplash.com/photo-1506354666786-959d6d497f1a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80"
             alt=""
           />
-          <h3 class="card-head__heading">Mr. Eat</h3>
+          <h3 class="card-head__heading">Bella Italia</h3>
         </div>
         <div class="card__card-body">
           <div class="card-body__left">
             <p class="card-body__text">
-              Bei Mr. Eat kriegt ihr die besten Falafel-Sandwiches in ganz Köln!
-              Mega günstig, mega zentral und mit etwas Glück gibt's eine Falafel
-              umsonst.
+              Kleiner, aber feiner italiener. So muss eine Pizzeria sein: klein,
+              gemütlich, einfach und lecker! Ohne Schnickschnack, große Auswahl,
```

Comment:
There are a few spelling mistakes.
"Kleiner, aber feiner **Italiener** " and  "**Preis-Leistungs-Verhältnis**".

I would also rewrite it a bit because "klein" and "lecker" are doubled.  Something like this:
"Kleiner, aber feiner Italiener mit einem super Preis-Leistungs-Verhältnis. Die Speisen werden schnell zubereitet und sind extrem lecker, kein Schnickschnack, eine tolle Auswahl und das Ambiente ist urgemütlich!"